cmd/bolt/main.go

```diff
@@ -562,6 +583,37 @@ func (cmd *PageCommand) PrintLeaf(w io.Writer, buf []byte) error {
 	return nil
 }
 
+// leafPageElement retrieves a leaf page element.
+func (cmd *PageCommand) leafPageElement(pageBytes []byte, index uint16) (*leafPageElement, error) {
+	p := (*page)(unsafe.Pointer(&pageBytes[0]))
+
+	if index >= p.count {
+		return nil, fmt.Errorf("Expected item index >= 0 and < %d, but got %d.", p.count, index)
```

Comment:
```go
fmt.Errorf("leafPageElement: expected item index less than %d, got %d", p.count, index)
```

to closer match the other `expected` errors and since index and p.count are unsigned